{"name": "test_a4", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "self = <ke21.test_case1.TestCase1 object at 0x0000025E3B1FECC0>\n\n    @allure.severity(\"critical\")\n    def test_a4(self):\n        print(\"2222222222222\")\n>       assert 1 == 2\nE       assert 1 == 2\n\nke21\\test_case1.py:28: AssertionError"}, "attachments": [{"name": "stdout", "source": "dccc47af-62eb-4302-a214-19743af2f5e2-attachment.txt", "type": "text/plain"}], "start": 1659146545209, "stop": 1659146545209, "uuid": "0be4627f-5e3d-4e83-9a7f-0342e5fa0c69", "historyId": "c44669749b6c79ea7b5683723ae4c4df", "testCaseId": "7106e3bda248994e0d7282fedc2d9b54", "fullName": "ke21.test_case1.TestCase1#test_a4", "labels": [{"name": "feature", "value": "添加商品"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "ke21"}, {"name": "suite", "value": "test_case1"}, {"name": "subSuite", "value": "TestCase1"}, {"name": "host", "value": "DESKTOP-7DB67S3"}, {"name": "thread", "value": "25720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "ke21.test_case1"}]}